includes/widgets/icon-box.php

```diff
@@ -653,27 +653,37 @@ protected function render() {
 		}
 		$migrated = isset( $settings['__fa4_migrated']['selected_icon'] );
 		$is_new = ! isset( $settings['icon'] ) && Icons_Manager::is_migration_allowed();
+
 		?>
 		<div class="elementor-icon-box-wrapper">
 			<?php if ( $has_icon ) : ?>
 			<div class="elementor-icon-box-icon">
-				<<?php echo implode( ' ', [ $icon_tag, $icon_attributes, $link_attributes ] ); ?>>
```

Comment:
Use `Utils::print_validated_html_tag` & `$this->print_render_attribute_string`
`